docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -93,17 +86,17 @@ exception items. Tags can also be used to define the {endpoint-artifact-name-sin
 |==============================================
 |Name |Type |Description |Required
 
-|`comments` |xref:exception-item-comment-schema[comment][] a|Array of xref:exception-item-comment-schema[`comment`] objects. |No, defaults to empty array.
+|`comments` |xref:exception-{endpoint-artifact-api-doc-name}-item-comment-schema[comment][] a|Array of xref:exception-{endpoint-artifact-api-doc-name}-item-comment-schema[`comment`] objects. |No, defaults to empty array.
 |`description` |String |Describes the {endpoint-artifact-name-single}. |Yes
-|`entries` |xref:exception-item-entry-schema[entry][] |Array containing the {endpoint-artifact-name-single} query conditions. See xref:exception-item-entry-schema[] for more details. |Yes.
+|`entries` |xref:exception-{endpoint-artifact-api-doc-name}-item-entry-schema[entry][] |Array containing the {endpoint-artifact-name-single} query conditions. See xref:exception-{endpoint-artifact-api-doc-name}-item-entry-schema[] for more details. |Yes.
 |`id` |String |The item's unique identifier. a|Yes, when the item's `item_id` field is not used.
 |`item_id` |String a|The `item_id` of the item you are updating. a|Yes, when the item's `id` field is not used.
 |`meta` |Object |Placeholder for metadata about the exception item. |No
 |`name` |String |The {endpoint-artifact-name-single} name. |Yes
 |`namespace_type` a|String a|Must be set to `agnostic`. |Yes
-|`os_types` |xref:exception-item-os-types[os_type][] |Array of xref:exception-item-os-types[]. | Yes
+|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Array of xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[]. | Yes
```

Comment:
I've updated `...item-os-types[]` to `item-os-types[os_types] values.` in my PR.